src/eval.c

```diff
@@ -1375,7 +1375,7 @@ char *ldbRedisProtocolToHuman_Double(sds *o, char *reply) {
 /* Log a RESP reply as debugger output, in a human readable format.
  * If the resulting string is longer than 'len' plus a few more chars
  * used as prefix, it gets truncated. */
-void ldbLogRedisReply(char *reply) {
+void ldbLogServerReply(char *reply) {
     sds log = sdsnew("<reply> ");
     ldbRedisProtocolToHuman(&log,reply);
```

Comment:
There are still a few "Redis" references in this file. Can we fix them all at once?
```suggestion
    ldbServerProtocolToHuman(&log,reply);
```

Reply:
I thought we decided to keep references to RedisProtocol? Are we renaming the protocol from RESP to something else?

Reply:
The protocol is not called "server protocol". It doesn't say much.

I suggested earlier (another PR) that we keep it as RedisProtocol here.

If we insist on changing it, we can change Redis to RESP here, i.e. name the function to ldbRespToHuman or ldbRespProtocolToHuman.

Reply:
Yeah, I would vote updating it to RESP, which feels more accurate.